mandatory/4-tax.js

```diff
@@ -17,7 +20,11 @@ function calculateSalesTax() {}
   Remember that the prices must include the sales tax (hint: you already wrote a function for this!)
 */
 
-function addTaxAndFormatCurrency() {}
+function addTaxAndFormatCurrency(number) {
+ let currency = calculateSalesTax(number);
```

Comment:
Is `currency` the best name for this variable here? What does the output of the function `calculateSalesTax` represent? Thinking of how to answer this question will help you to improve your variable name.